src/explore-education-statistics-frontend/src/modules/table-tool/components/PublicationForm.tsx

```diff
@@ -183,7 +185,7 @@ const PublicationForm = (props: Props & InjectedWizardProps) => {
                 {stepHeading}
                 <SummaryList noBorder>
                   <SummaryListItem term="Publication">
-                    {publicationName}
+                    {publicationName || publicationTitle}
```

Comment:
If we're just passing in the publication title from above this component, then we don't really need `publicationName` anymore.

Reply:
@ntsim I was doing this because after being passed through the publicationName can still be changed by a user selecting a different publication. However, publicationTitle cant be reassigned if this happens because the props get passed through as a const and if I change it to let then linting blows up.

Reply:
`publicationName` is only important once the user submits the form, at which point it passes the `publicationId` back up to `TableToolPage`, which then sets `publication`. This means we _should_ have `publication` available to pass back down to `PublicationForm` to display the title from (using `publicationTitle`).

Would just get rid of the `publicationName` state.

And also agree that you shouldn't be changing it to let. That's really bad practice.

Reply:
Ahh okay I get you. I'll make the change :)